web/pages/home/index.tsx

```diff
@@ -324,6 +347,86 @@ function SearchSection(props: {
   )
 }
 
+function FeaturedSection() {
+  const [pinned, setPinned] = useState<JSX.Element[]>([])
+  const [posts, setPosts] = useState<Post[]>([])
+  const [homeConfig, setHomeConfig] = useState<HomeConfig | undefined>(
+    undefined
+  )
+  useEffect(() => {
+    getHomeConfig().then(setHomeConfig)
+    getAllPosts().then(setPosts)
+  }, [])
```

Comment:
This would only start loading when the component is rendered. The other sections load above from hooks.

One way to improve this is to extract this loading into a hook, load the data above, and pass it down into the section.

Another idea would be to load these in the getStaticProps, although that won't always be fresh data, e.g. if you just edited the pinned items.